### PR TITLE
chore: remove prettier, and default to eslint

### PR DESCRIPTION
**Pull request by danisharora099**
This PR 
- removes prettier altogether
- uses eslint for formatting + linting
- sets vscode default formatter to eslint, from prettier
- reruns `npm run proto`
   - since we don't run `prettier` on it anymore, there is a huge diff
   - ideally, we should remove `proto` from `eslintIgnore` but i am not sure why it was added to ignore (https://github.com/waku-org/js-waku/commit/704f2770d1d4aac64726071bad2a22ef7aa84e66)

### CODE REVIEW

**Review by fryorcraken — CHANGES_REQUESTED** [review]
Do not run format or lint on generated files:
- it just creates more mess as you know have 2 scripts that breaks each-other outputs (same as you had with eslint and prettier)
- the files are auto-generated so nobody is going to "mess up" the formatting
- the files are auto-generated so even if the linter find an issue, there is nothing you will be able to do to fix it.

File: .eslintrc.json [hunk 1]
```diff
@@ -68,19 +64,10 @@
...
     {
       "files": ["*.spec.ts", "**/test_utils/*.ts", "*.js", "*.cjs"],
-      "env": {
-        "node": true
-      },
       "rules": {
         "@typescript-eslint/no-non-null-assertion": "off",
         "@typescript-eslint/no-explicit-any": "off",
-        "no-console": "off",
-        "import/no-extraneous-dependencies": [
```
Comment (fryorcraken) [inline .eslintrc.json:78]: Why removing this?

File: .eslintrc.json [hunk 2]
```diff
@@ -1,9 +1,6 @@
 {
   "root": true,
   "parser": "@typescript-eslint/parser",
-  "parserOptions": {
-    "project": ["./tsconfig.json"]
```
Comment (fryorcraken) [inline .eslintrc.json:5]: Why removing this?

File: .eslintrc.json [hunk 3]
```diff
@@ -1,9 +1,9 @@
 {
   "root": true,
   "parser": "@typescript-eslint/parser",
-  "parserOptions": {
+   "parserOptions": {
```
Comment (weboko) [inline .eslintrc.json:4]: probably you need to re-run prettier/linter

File: packages/proto/src/lib/filter.ts [hunk 4]
```diff
@@ -4,542 +4,511 @@
...
+        _codec = message<ContentFilter>((obj, w, opts = {}) => {
+          if (opts.lengthDelimited !== false) {
+            w.fork()
+          }
 
-            if (obj.contentTopic != null && obj.contentTopic !== "") {
-              w.uint32(10);
-              w.string(obj.contentTopic);
-            }
+          if ((obj.contentTopic != null && obj.contentTopic !== '')) {
```
Comment (weboko) [inline packages/proto/src/lib/filter.ts:32]: oh no, why do we change to single quote?

**Review by fryorcraken — APPROVED** [review]

**Pull request merged**